Quasardb.Native/qdb_string.cs

```diff
@@ -8,9 +8,10 @@ namespace Quasardb.Native
     public unsafe struct qdb_string
     {
         public byte* Chars;
-        
+
         public override string ToString()
         {
+            if (Chars == null) return String.Empty;
             // TODO: should be UTF8, not ANSI
             return Marshal.PtrToStringAnsi(new IntPtr(Chars));
```

Comment:
`PtrToStringUTF8`?
https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.marshal.ptrtostringutf8?view=netcore-3.0&viewFallbackFrom=netframework-4.8

Reply:
I don't know, this is existing code. What could have been the rationaly for not using it back then?

Reply:
This method is not available for the project. It appears not to be included in the .NET Framework

Reply:
Possibly an omission I guess. You may want to do it in a separate commit tho.